tests/rptest/tests/compaction_end_to_end_test.py

```diff
@@ -75,6 +75,7 @@ def topic_segments(self):
             ],
             transactions=[True, False],
             tx_inject_aborts=[True, False])
+    @skip_debug_mode
```

Comment:
@mmaslankaprv Is this change intentional? The commit message says something else.